src/forms-opportunities/utils.js

```diff
@@ -94,43 +94,45 @@ function getFormMetrics(metricObject) {
 function convertToLowViewOpptyData(metricObject) {
   const {
     formview: { total: formViews, mobile: formViewsMobile, desktop: formViewsDesktop },
-    trafficacquisition,
+    pageview: { total: pageViews, mobile: pageViewsMobile, desktop: pageViewsDesktop },
+    // trafficacquisition,
   } = metricObject;
   return {
-    trackedFormKPIName: 'Form Views',
+    trackedFormKPIName: 'Form View Rate',
     trackedFormKPIValue: formViews,
     metrics: [
       {
-        type: 'formViews',
+        type: 'formViewRate',
         device: '*',
         value: {
-          page: formViews,
+          page: formViews / pageViews,
         },
       },
       {
-        type: 'formViews',
+        type: 'formViewRate',
         device: 'mobile',
         value: {
-          page: formViewsMobile,
+          page: formViewsMobile / pageViewsMobile,
         },
       },
       {
-        type: 'formViews',
+        type: 'formViewRate',
         device: 'desktop',
         value: {
-          page: formViewsDesktop,
-        },
-      },
-      {
-        type: 'traffic',
-        device: '*',
-        value: {
-          total: trafficacquisition.total ? trafficacquisition.total : null,
-          paid: trafficacquisition.paid ? trafficacquisition.paid : null,
-          earned: trafficacquisition.earned ? trafficacquisition.earned : null,
-          owned: trafficacquisition.owned ? trafficacquisition.owned : null,
+          page: formViewsDesktop / pageViewsDesktop,
         },
       },
+      // Enable when trafficacquisition data in enabled from spacecat-shared
```

Comment:
lets have the following data for every oppoty type: (that doesn't needs `trafficacquisition` and can be calculated from pageViews itself)
```
 {
      type: 'traffic',
      device: 'desktop/mobile',
      value: {
        total: trafficacquisition.total ? trafficacquisition.total : null,
    },
}
```

Reply:
Is there any reason for metrics to be an array ?
can it not be 
```
metrics : {
   [metricname]: {
       [device]: {value: ..., any other relevant data}
   }
}
```

Reply:
no, I guess just consistency with sites metrics schema that helps us with UI code too.